docs/release-notes/6701-deprecate-estimators.rst

```diff
@@ -0,0 +1,7 @@
+:orphan:
+
+**Deprecated Features**
+
+-  ``EstimatorTrial`` has been deprecated and will be removed in a future version. TensorFlow
+   has recommended Estimator users switch to Keras since the release of TensorFlow 2.0, and so
+   Determined recommends EstimatorTrial users switch to :class:`~determined.keras.TFKerasTrial`.
```

Comment:
EstimatorTrial is deprecated and will be removed in a future version. TensorFlow has advised Estimator users to switch to Keras since TensorFlow 2.0 was released. Consequently, we recommend users of EstimatorTrial to switch to the :class:`~determined.keras.TFKerasTrial` class.